src/os/bluestore/BlueFS.cc

```diff
@@ -3563,7 +3557,8 @@ int BlueFS::lock_file(std::string_view dirname, std::string_view filename,
     file_map[ino_last] = file;
     dir->file_map[string{filename}] = file;
     ++file->refs;
-    log_t.op_file_update(file->fnode.make_full());
+    log_t.op_file_update_inc(file->fnode);
```

Comment:
File is created at this point hence it should be full update IMO

Reply:
True.